docs/build-instructions/windows.md

```diff
@@ -31,8 +31,14 @@ Note: If you use cmd or Powershell instead of Git Shell, use a backslash instead
 These instructions will assume the use of Git Shell.
 
 ### `script/build` Options
-  * `--install-dir` - Creates the final built application in this directory.
-  * `--build-dir` - Build the application in this directory.
+  * `--install-dir` - Creates the final built application in this directory. Example (trailing slash is optional):
+```bash
+./script/build --install-dir Z:\Some\Destination\Directory\
```

Comment:
We actually found that when opening the shell from Github Desktop (so, the Git Shell), `script\build` works, `script/build` does not work, `./script/build` does not work either. Thinking that it was just us, we did not changed the convention found in the doc page.

---
On a totally different topic: 
This PR just affects an .md file, still Travis CI reports (random) errors similar to:

```
Unable to download https://www.atom.io/api/packages/<some package name, version, etc>

The command "script/cibuild" exited with 1.
```

where the *random* part is related to which package actually fails to download.